snippets/logging-pruning-recc-snip.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in the following assemblies:
+//
+// 
+//
+// Text snippet included in the following modules:
+//
+//
+
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
+====
+For cost-efficient logs pruning, it is recommended to set global retention on the object storage provider side as this does not divert Loki's resources. Pruning older logs is a cost free function provided by the storage provider.
```

Comment:
1. **suggestion:** s/logs pruning/log pruning
2. **suggestion:** Add comma `provider side, as this does`
3. **nitpick:** Remove `cost` s/cost free/free

Reply:
`For cost-efficient log pruning, it is recommended to set global retention on the object storage provider side, as this does not divert Loki's resources. Pruning older logs is a free function provided by the storage provider.`